_pages/pages/documentation/external-tools-and-resources.md

```diff
@@ -0,0 +1,68 @@
+---
+title: External Tools and Resources
+permalink: /pages/documentation/external-tools-and-resources/
+layout: docs
+navigation: pages
+sidenav: pages-documentation
+---
+
+An outline of external tools and resources customer could leverage outside of the cloud.gov Pages platform to improve site accessibility, user experience, and security posture.
+
+Cloud.gov Pages sites' source code is managed via [Git](https://git-scm.com/) version control and hosted on [GitHub](https://www.github.com). The combination of these source code management tools allows cloud.gov Pages customers to leverage open source tools, third-party services, and other resources outside of the cloud.gov Pages platform to improve their development workflow, user experience, and security posture. This will outline different areas of interest with possible resources for customers to research in-depth based on their current priorities, resources, skill sets, and constraints. This is not an endorsement of any single resource or tool so this outline will change as with the landscape and it is only a small sample of possible tools and resources.
```

Comment:
```suggestion
Cloud.gov Pages sites' source code is managed via [Git](https://git-scm.com/) version control and hosted on [GitHub](https://www.github.com). The combination of these source code management tools allows cloud.gov Pages customers to leverage open source tools, third-party services, and other resources outside of the cloud.gov Pages platform to improve their development workflow, user experience, and security posture. This will outline different areas of interest with possible resources for customers to research in-depth based on their current priorities, resources, skill sets, and constraints. This is not an endorsement of any single resource or tool so this outline will change with the landscape and it is only a small sample of possible tools and resources.
```